CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Overview
+This page sets forth the guidelines and policies for continued development. If you are a new developer, you are welcome to contribute. Read everything here and you shoud feel comfortable contributing code.
+
+If you would like to get started helping, please read everything here and follow the steps under [creating a pull request](#pullreq).
+
+
+# Table of Contents
+* [Dependencies](#dependencies)
+* [Branches & versioning](#branches-versioning)
+* [Commit guidelines](#commits)
+* [Creating a pull request](#pullreq)
+
+# Dependencies
+In order to contribute code to this project, install these prerequisites. If you are using a Linux distribution, search your package manager's repository for the applicable packages.
+- [.NET Core 2.0+ SDK and Runtime](https://dotnet.microsoft.com/download)
+- [Visual Studio Code or Community 2017+](https://visualstudio.microsoft.com/downloads/)
+
```

Comment:
i would  split this up into two sections, the "easy" one and the "manual" steps
## "easy" way using docker
- [Docker](https://www.docker.com/get-started)
bla bla 
```bash
docker build objpatrish:latest .
docker run -d -v /src:/app objpatrish:latest
```
## manual local setup
- [.NET Core 2.0+ SDK and Runtime](https://dotnet.microsoft.com/download)
- [Visual Studio Code or Community 2017+](https://visualstudio.microsoft.com/downloads/)